### ISSUE TITLE
Issue delegating to properties

### ISSUE THREAD

**Issue by valveca-ai** [dhilst/delegate#1]
When trying to delegate to a property, and using the getter function, the returned value to Python is the bound method instead of the property value.

Example:
```
    >>> class Bar:
    ...     def __init__(self):
    ...         self._param = 0
    ...     @property
    ...     def param(self):
    ...         return self._param
    ...     @param.setter
    ...     def param(self, param):
    ...         self_param = param

    >>> @delegate('v', 'param')
    ... class Foo2:
    ...     def __init__(self):
    ...         self.v = Bar()

    >>> foo2 = Foo2()
    >>> foo2.param
```

A `0` is expected, but this is returned:
```
<bound method delegate.<locals>.dec.<locals>.create_delegator.<locals>.delegator of
<__main__.Foo2 object at 0x100c07490>>
```

A fix has been made and will be sent via a PR

**Comment by dhilst** [comment]
Hii!!
Thanks again for contributing, I just updated the pip package with #2 updates, `pip install --update delegateto` should take the fix

Regards!

**Comment by valveca-ai** [comment]
Updating properties was not completely implemented in PR #2: the property was not being properly set in the delegated object. The fix and updated README file has been included in a new PR. 

This new test case was failing:

```
    >>> foo2 = Foo2()
    >>> foo2.param
    0
    >>> foo2.param = 2
    >>> foo2.param
    2
    >>> foo2.v.param
    2
    >>> foo2.v._param
    2
```
 A `2` was expected in the last two cases but `0` was being returned.
